docs/pages/2020_Intro_Week/index.rst

```diff
@@ -23,11 +23,75 @@ This is the content of the `AiiDA virtual tutorial <http://www.aiida.net/aiida-v
++--------------------------+----------------------------------------------------------------------------------------------------------------------+
+| Dominik Gresch           | `Introduction to aiida-optimize`_                                                                                    |
++--------------------------+----------------------------------------------------------------------------------------------------------------------+
+| Atsushi Togo             | `Harmonic phonon calculation using AiiDA AiiDA-VASP AiiDA-phonopy`_                                                  |
++--------------------------+----------------------------------------------------------------------------------------------------------------------+
+| Bonan Zhu                | `Introducing aiida-castep`_                                                                                          |
++--------------------------+----------------------------------------------------------------------------------------------------------------------+
+| Emanuele Bosoni          | `Introducing aiida-siesta`_                                                                                          |
++--------------------------+----------------------------------------------------------------------------------------------------------------------+
+| Philipp Rüßmann          | `The aiida-kkr plugin`_                                                                                              |
++--------------------------+----------------------------------------------------------------------------------------------------------------------+
+| Miki Bonacci             | `The Yambo-AiiDA plugin`_                                                                                            |
++--------------------------+----------------------------------------------------------------------------------------------------------------------+
+
+.. _Using AiiDA to study gas adsorption in microporous crystals: https://www.youtube.com/watch?v=9p_krkdEz_A&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _Introduction to aiida-optimize: https://www.youtube.com/watch?v=YzHuIrYhyFI&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _Harmonic phonon calculation using AiiDA AiiDA-VASP AiiDA-phonopy: https://www.youtube.com/watch?v=Uqz-i29JD5U&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _Introducing aiida-castep: https://www.youtube.com/watch?v=X18SiQwVPQo&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _Introducing aiida-siesta: https://www.youtube.com/watch?v=0S7rvut8gpE&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _The aiida-kkr plugin: https://www.youtube.com/watch?v=ccmEWKgVfVc&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _The Yambo-AiiDA plugin: https://www.youtube.com/watch?v=NzhN5ce0Sc8&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+
+Many thanks to the plugin developers for their contribution!
```

Comment:
Nice, but not right at this location, I think? Maybe under acknowledgements instead?

Reply:
Moved to acknowledgements!